pom.xml

```diff
@@ -1,9 +1,9 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
-  <groupId>com.axonivy.connector.currencyr</groupId>
+  <groupId>com.axonivy.connector.currency</groupId>
```

Comment:
uh. 9.2.0 released under the wrong groupId?

Reply:
Only the module pom. And this is not deployed right, so no problem...